packages/js/src/dashboard/store/alert-center.js

```diff
@@ -0,0 +1,116 @@
+/* global ajaxurl */
```

Comment:
Any ideas how to not rely on this global? 🤔

Reply:
moved it to the store